python/pyfunc-server/setup.py

```diff
@@ -13,19 +13,23 @@
 # limitations under the License.
 
 import os
-from setuptools import setup, find_packages
+
+from setuptools import find_packages, setup
 
 tests_require = [
-    'pytest',
-    'pytest-tornasync',
-    'requests',
-    'types-requests',
-    'types-protobuf',
-    'mypy',
-    'pytest-benchmark',
+    "joblib>=0.13.0,<1.2.0",  # >=1.2.0 upon upgrade of kserve's version
+    "mypy",
+    "pytest-benchmark",
+    "pytest-tornasync",
+    "pytest",
+    "requests",
+    "scikit-learn>=1.1.2",
+    "types-protobuf",
+    "types-requests",
+    "xgboost==1.6.2",
```

Comment:
why is this pinned suddenly?

Reply:
I added `test_examples.py` which will build and run the Iris PyFunc model locally. This model requires joblib, sklearn, and xgboost.